doc/release-notes/release-notes-bucash1.2.0.0.md

```diff
@@ -0,0 +1,139 @@
+Release Notes for Bitcoin Unlimited Cash Edition 1.2.0.0
+=========================================================
+
+Bitcoin Unlimited Cash Edition version 1.2.0.0 is now available from:
+
+  <https://bitcoinunlimited.info/download>
+
+Please report bugs using the issue tracker at github:
+
+  <https://github.com/BitcoinUnlimited/BitcoinUnlimited/issues>
+
+This is a minor release version based of Bitcoin Unlimited compatible
+with the Bitcoin Cash specification you could find here:
+
```

Comment:
isn't this a major release?

Reply:
you could be right, I always forgot the semantics of the various version level. 
@gandrewstone I think @ptschip is right. what do you say?

Reply:
minor actually.  the semantics are: major.minor.periodic_bugfix.hotfix
Regardless of what you call the last 2, the first 2 being "major.minor" is pretty universally agreed upon.

Reply:
ok